Src/Microsoft.Scripting/Runtime/DynamicOperations.cs

```diff
@@ -155,8 +155,7 @@ public sealed partial class DynamicOperations {
         /// Gets the member name from the object obj.  Throws an exception if the member does not exist or is write-only.
         /// </summary>
         public object GetMember(object obj, string name, bool ignoreCase) {
-            CallSite<Func<CallSite, object, object>> site;
-            site = GetOrCreateSite<object, object>(_lc.CreateGetMemberBinder(name, ignoreCase));
+            CallSite<Func<CallSite, object, object>> site = GetOrCreateSite<object, object>(_lc.CreateGetMemberBinder(name, ignoreCase));
```

Comment:
Ah, the eternal var debate. I agree that in this case it's not clear what the method returns, but I also agree that it's a long and ugly type. We should probably come up with some guidelines..